GERTi/GERTiClient.lua

```diff
@@ -154,45 +230,60 @@ end
 
 -- opens a route using the given information, used in handler["OPENROUTE"] and GERTi.openRoute()
 -- DIFFERs from MNC's orController
-local function orController(destination, origination, beforeHop, nextHop, receivedPort, transmitPort, ...)
+-- 'cb' (callback) is given success (true) or failure (false)
+-- The ... is sent in the MNC's direction
+local function orController(destination, origination, beforeHop, nextHop, receivedPort, transmitPort, cb, ...)
```

Comment:
cb is always nil, consider removing?

Reply:
It, in fact, is not always nil - see GERTi.openRoute.